nodejs/advanced/event-detection-ui/README.md

```diff
@@ -0,0 +1,28 @@
+# Event Detection Demo UI
+
+This is the UI for the event detection demo. Stream data from your phone or a data generator and see the telemetry in this UI in real-time. Run an event detection service and you can also see event time and location too.
+
+## Docs
+
+Check out the [API docs](https://docs.quix.io/apis/streaming-reader-api/intro/) for detailed API usage guidance and how to connect to the WebSockets API using SignalR.
+
+## How to run in the Quix Portal
+
+Create an account on [Quix](https://portal.platform.quix.ai/self-sign-up?xlink=github) to edit or deploy this application without a local environment setup.
+
+### Local development
```

Comment:
Wasn't sure this should be an h3? I think an h2 is fine, as it is not a sub-topic from what I can tell.